packages/create-sitecore-jss/src/templates/nextjs-personalize/src/lib/component-props/index.ts

```diff
@@ -0,0 +1,4 @@
+﻿// NULL means Hidden by this experience
```

Comment:
You'll need to copy the base file content here as well (there is no "merge", just overwriting).